consensus/state_test.go

```diff
@@ -310,14 +310,21 @@ func TestStateOversizedBlock(t *testing.T) {
 				totalBytes += len(part.Bytes)
 			}
 
+			maxBlockParts := maxBytes / int64(types.BlockPartSizeBytes)
+			if maxBytes > maxBlockParts*int64(types.BlockPartSizeBytes) {
+				maxBlockParts += 1
+			}
+			numBlockParts := int64(propBlockParts.Total())
```

Comment:
```suggestion
			maxBlockParts := (maxBytes-1) / int64(types.BlockPartSizeBytes) + 1
			numBlockParts := int64(propBlockParts.Total())
```

How about this instead?

Reply:
I prefer using a different method, even if it is less efficient/smart, when testing.

Reply:
Good point